index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <link rel="stylesheet" href="css/style.css">
+  <body>
+    <nav id="navbar"><h1>  Firts Navbar</h1>
+      <ul>
+        <li><a href="#welcome-section">Welcome</a></li>
+        <li><a href="#projects">Projects</a></li>
+      </ul>
+    </nav>
+    <a href="https://github.com/AntonioHincapie/Muhammad_Marco_CollaborativeWork.git" target="_blank"  id="profile-link"><h4>GitHub</h4></a>
+   
+    <div id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <h2>Lorem Imspum!</h2>
+    </div>
+    <section id="projects">
+      <div id="image">
+        <img src="img/landscape.jpg" alt="Beatiful Landscape">
+      </div>
+      <div class="project-tile">
+       <a href="https://codepen.io/freeCodeCamp/full/zNBOYG">Project Link</a> 
+      </div>
+    </section>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Scripts should be maintained in a separate file

Reply:
@for-ashraf It would be better if you explain why it should be , it may be obvious for you but not for everyone so please try to do that